### ISSUE TITLE
API fatal error handler returned after process out of memory

### ISSUE THREAD

**Issue by xyproto** [marktext/marktext#486]
### Description

marktext crashes when opening a markdown file

### Steps to reproduce

1. Install marktext
2. Download https://raw.githubusercontent.com/xyproto/algernon/master/README.md
3. Run `marktext README.md`

**Expected behavior:**

To open the file and show the contents.

**Actual behavior:**

```
#
# Fatal error in , line 0
# API fatal error handler returned after process out of memory
#
#0 0x000004685997 <unknown>
#1 0x0000026e1dd5 <unknown>
#2 0x7fa82e55935d V8_Fatal()
#3 0x7fa82df6b533 <unknown>
#4 0x7fa82e037e9e <unknown>
#5 0x7fa82e022091 <unknown>
#6 0x7fa82e021961 <unknown>
#7 0x7fa82e021823 <unknown>
#8 0x7fa82e03d625 <unknown>
#9 0x7fa82e02fa9a <unknown>
#10 0x00000467429b <unknown>
#11 0x000004673bee <unknown>
#12 0x000004677540 <unknown>
#13 0x00000467389c <unknown>
#14 0x00000466fe49 <unknown>
#15 0x0000046281e3 <unknown>
#16 0x7fa82d67ea9d start_thread
#17 0x7fa828d26a43 __GI___clone
```

### Versions

- Mark Text: v 0.12.25
- OS: Arch Linux

**Comment by fxha** [comment]
I think this is a duplicate of #481. As a workaround you can drag&drop or open the file via menu.

**Comment by xyproto** [comment]
It's hard to tell if it's a duplicate, since #481 includes no terminal output or error messages.

**Comment by fxha** [comment]
I'm closing this issue since it seems to be resolved by PR #483. I tested the following successfully:

```
./marktext GIVEN_README.md
```

The problem was that you cannot launch Mark Text with command line arguments. Please reopen if I missed something.
